File: frontend/web_ccadmin_lt/src/app/enterprise/product/model/dto/KardexZoneDto.ts
import { KardexZoneEntity } from "../entity/KardexZoneEntity";
import { ProductEntity } from "../entity/ProductEntity";

export class KardexZoneDto {
    public kardexZone: KardexZoneEntity = new KardexZoneEntity();
    public product: ProductEntity = new ProductEntity();
}

File: backend/ws_wa_store_ccadmin/src/main/java/com/ccadmin/app/product/model/dto/KardexZoneDto.java
package com.ccadmin.app.product.model.dto;

import com.ccadmin.app.product.model.entity.KardexZoneEntity;
import com.ccadmin.app.product.model.entity.ProductEntity;

public class KardexZoneDto {

    public KardexZoneEntity kardexZone;
    public ProductEntity product;

    public KardexZoneDto(KardexZoneEntity kardexZone, ProductEntity product) {
        this.kardexZone = kardexZone;
        this.product = product;
    }
}
